### PR TITLE
Fix nil panic

### PR DESCRIPTION
**Pull request by xwduan**
**What changed?**
Add nil check when using namespaceEntry

**Why?**
nil panic found at edge case

**How did you test it?**
n/a

**Potential risks**
no

**Is hotfix candidate?**
no

### CODE REVIEW

**Review by wxing1292 — APPROVED** [review]

File: service/history/replication/raw_task_converter.go [hunk 1]
```diff
@@ -116,8 +116,15 @@ func (c *SourceTaskConverterImpl) Convert(
 	if clientShardID != c.clientShardKey.ShardID {
 		return nil, nil
 	}
+	var ctx context.Context
+	var cancel context.CancelFunc
+
+	if namespaceEntry != nil {
+		ctx, cancel = newTaskContext(namespaceEntry.Name().String())
```
[the remark this answers is not in the document]
Comment (xwduan) [reply service/history/replication/raw_task_converter.go:123]: Yeah, I think we can. But I am a little bit reluctant to either change the input of `newTaskContext` function or pass an empty string...
